packages/grid/x-data-grid/src/locales/esES.ts

```diff
@@ -71,33 +71,33 @@ const esESGrid: Partial<GridLocaleText> = {
   filterOperatorIsEmpty: 'está vacío',
   filterOperatorIsNotEmpty: 'no esta vacío',
   filterOperatorIsAnyOf: 'es cualquiera de',
-  // 'filterOperator=': '=',
-  // 'filterOperator!=': '!=',
-  // 'filterOperator>': '>',
-  // 'filterOperator>=': '>=',
-  // 'filterOperator<': '<',
-  // 'filterOperator<=': '<=',
+  'filterOperator=': 'es igual a',
+  'filterOperator!=': 'es diferente a',
+  'filterOperator>': 'es mayor que',
+  'filterOperator>=': 'es mayor o igual que',
+  'filterOperator<': 'es menor que',
+  'filterOperator<=': 'es menor o igual que',
 
   // Header filter operators text
-  // headerFilterOperatorContains: 'Contains',
-  // headerFilterOperatorEquals: 'Equals',
-  // headerFilterOperatorStartsWith: 'Starts with',
-  // headerFilterOperatorEndsWith: 'Ends with',
-  // headerFilterOperatorIs: 'Is',
-  // headerFilterOperatorNot: 'Is not',
-  // headerFilterOperatorAfter: 'Is after',
-  // headerFilterOperatorOnOrAfter: 'Is on or after',
-  // headerFilterOperatorBefore: 'Is before',
-  // headerFilterOperatorOnOrBefore: 'Is on or before',
-  // headerFilterOperatorIsEmpty: 'Is empty',
-  // headerFilterOperatorIsNotEmpty: 'Is not empty',
-  // headerFilterOperatorIsAnyOf: 'Is any of',
-  // 'headerFilterOperator=': 'Equals',
-  // 'headerFilterOperator!=': 'Not equals',
-  // 'headerFilterOperator>': 'Greater than',
-  // 'headerFilterOperator>=': 'Greater than or equal to',
-  // 'headerFilterOperator<': 'Less than',
-  // 'headerFilterOperator<=': 'Less than or equal to',
+  headerFilterOperatorContains: 'Contiene',
+  headerFilterOperatorEquals: 'Es igual a',
+  headerFilterOperatorStartsWith: 'Comienza con',
+  headerFilterOperatorEndsWith: 'Termina con',
+  headerFilterOperatorIs: 'Es',
+  headerFilterOperatorNot: 'No es',
+  headerFilterOperatorAfter: 'Esta despues de',
+  headerFilterOperatorOnOrAfter: 'Esta en o después de',
+  headerFilterOperatorBefore: 'Esta antes de',
+  headerFilterOperatorOnOrBefore: 'Esta en o antes de',
+  headerFilterOperatorIsEmpty: 'Esta vacio',
+  headerFilterOperatorIsNotEmpty: 'No esta vacio',
+  headerFilterOperatorIsAnyOf: 'Es cualquiera de',
+  'headerFilterOperator=': 'es igual a',
+  'headerFilterOperator!=': 'es diferente a',
+  'headerFilterOperator>': 'es mayor que',
+  'headerFilterOperator>=': 'es mayor o igual que',
+  'headerFilterOperator<': 'es menor que',
+  'headerFilterOperator<=': 'es menor o igual que',
```

Comment:
I think these need to be capitalized like their English counterparts to be consistent.

Reply:
Yep bro, that's right, thanks for feedback, already corrected!

Reply:
@Macampu420 like this:

![image](https://github.com/mui/mui-x/assets/1423607/1d81a3bd-66fa-4723-ad06-3e6d00fc02f9)